src/helpers/helper.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { TypeOfFilter } from '../types/TypeOfFilters';
+
+export const filterTodos = (
+  todos: Todo[],
+  filter: TypeOfFilter,
+) => {
+  let copy = [...todos];
```

Comment:
`copy` is very common name, also filter does not mutate array, you can filter your todos directly